Base/QTGUI/qSlicerSettingsModulesPanel.cxx

```diff
@@ -161,7 +161,7 @@ void qSlicerSettingsModulesPanelPrivate::init()
                       "showModules", SIGNAL(showModulesChanged(QStringList)));
   qSlicerRelativePathMapper* relativePathMapper = new qSlicerRelativePathMapper(
     this->TemporaryDirectoryButton, "directory", SIGNAL(directoryChanged(QString)));
-  q->registerProperty("Modules/TemporaryDirectory", relativePathMapper,
+  q->registerProperty("TemporaryPath", relativePathMapper,
```

Comment:
I suggest we add an entry to  https://www.slicer.org/wiki/Documentation/Nightly/Developers/Tutorials/MigrationGuide. 

For example, a category called `Slicer 5.0: API changes since 4.10 / Settings` could  be added.

Reply:
Added https://www.slicer.org/wiki/Documentation/Nightly/Developers/Tutorials/MigrationGuide/Slicer#Slicer_5.0_:_Temporary_path. If we have more settings changes then we may merge them under a Settings section.

Reply:
Thanks. I tweaked formatting using `<code>...</code>`